.github/pull_request_template.md

```diff
@@ -1,15 +1,15 @@
 # Trac ticket number
-<!-- Replace [number] with the corresponding Trac ticket number, or delete the line and write "N/A" if this is a trivial PR. -->
+<!-- Replace XXXXX with the corresponding Trac ticket number, or delete the line and write "N/A" if this is a trivial PR. -->
 
-ticket-[number]
+ticket-XXXXX
```

Comment:
This change is driven by a few contributors that have understood that they should only replace the word `number` and would leave the square brackets unremoved (thus the ticket link does not work :sob: )